### DISCUSSION TITLE
Development on Linux?

### DISCUSSION THREAD

**Discussion by Sigurdur42** [unoplatform/uno#10027] [Q&A]
Good morning,
Is someone actively developing on Linux? If so, which distro is recommended?

Or are you all developing on Windows and just publish for Linux? Or are you using WSL somehow?

I really want to switch to Linux but development ist the last thing really in my way...

Thx

**Comment by jeromelaban** [comment]
At this time, our testing is mostly done on [Debian-based distros](https://platform.uno/docs/articles/get-started-with-linux.html?tabs=ubuntu2004%2Cwindows) as you can see in our docs. You can also use [GitPod for development](https://platform.uno/docs/articles/features/working-with-gitpod.html).

If you want to build for windows (Skia.Wpf or Windows heads), you'll need to build on Windows for now as the underlying tooling only works on Windows.

**Reply by HeyMerlin** [reply]
A further question about setting up for Linux development. On the page you link for Debian-based distros:
- VS Code section indicates .Net 6.0 sdk or later
- Linux section specifies to install .Net sdk 3.1 and 5.0

I'm trying to figure out how many versions of the .Net sdk I need to install...

Are there updated requirements for Ubuntu 22.04?

Are both versions 3.1 and 5.0 still required to be installed?

Thanks.

**Reply by jeromelaban** [reply]
Indeed, the documentation needs updating. Using .NET 6 is fine, and soon .NET 7.
